core/cas-server-core-services/src/test/java/org/apereo/cas/util/services/RegisteredServiceJsonSerializerTests.java

```diff
@@ -30,5 +31,23 @@ public void checkNullability() {
         assertNotNull(s.getAttributeReleasePolicy());
         assertNotNull(s.getProxyPolicy());
         assertNotNull(s.getUsernameAttributeProvider());
+        assertNotNull(s.getLogoutType());
+    }
+
+    @Test
+    public void checkLogoutType() {
+        final RegisteredServiceJsonSerializer zer = new RegisteredServiceJsonSerializer();
+        final String json = "{    \n"
```

Comment:
Please update this test so that:

1. Create a service object with the relevant properties
2. Save it as JSON, serializing it.
3. Then read the same file again as JSON and ensure it matches what you expect.

Do not hardcode raw JSON snippets into tests.

Otherwise, excellent.